examples/QtSiteConfig/QtSiteConfig.py

```diff
@@ -22,6 +27,15 @@ def update_misplaced_members(members):
     Arguments:
         members (dict): The members considered by Qt.py
     """
+    # Printer support in Qt4 was stored under QtGui instead of QtPrintSupport
+    members['PyQt4']['QtGui.QPrintPreviewWidget'] = \
```

Comment:
Could we stick with double-ticks for quotation marks, and parenthesis for line-breaks?

E.g.

```python
members["PyQt4"]["QtGui.QPrintPreviewWidget"] = (
  "QtPrintSupport.QPrintPreviewWidget"
)
```

And then similarly for assertions.

```python
name = QtPrintSupport.QPrintPreviewWidget.__name__
assert name == "QPrintPreviewWidget', (
  "Qt.QtPrintSupport.QPrintPreviewWidget was not correctly mapped"
)
```

Reply:
Oops, I try to use double-ticks for Qt.py, but missed those.
I was trying to figure out how to do a multi-line assert as that line was extremely long but assert's don't work as functions. I had not thought of using the parenthesis like that before.